lib/DocumentCard/DocumentCard.js

```diff
@@ -92,7 +92,7 @@ class DocumentCard extends React.Component {
           <Col xs={category ? 8 : 12}>
             {note &&
               <KeyValue label={<FormattedMessage id="stripes-erm-components.doc.note" />}>
-                <div data-test-doc-note>{note}</div>
+                <div data-test-doc-note style={{ whitSpace: 'pre-wrap' }}>{note}</div>
```

Comment:
`whitSpace`.

Can you run through and actually test all of these to make sure everything works correctly?

Reply:
oops! tested and all tests pass.